RubberduckTestsCodeAnalysis/RubberduckTestsCodeAnalysis.csproj

```diff
@@ -1,242 +1,160 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
FYI for some reason the project isn't listed in the solution's object explorer.

Reply:
This is fixed  with 74f09ba